src/main/java/com/team701/buddymatcher/dtos/pairing/AddBuddyDTO.java

```diff
@@ -4,20 +4,20 @@
  * Data Transfer Object representing a request to add a buddy to a user
  */
 public class AddBuddyDTO {
-    private String userId;
-    private String buddyId;
+    private Long userId;
```

Comment:
These pairing DTOs were refactored after changing the User id from UUID to Long. The integration test began failing once I rebased off the main repository.

Reply:
Good work for refactoring those!